samples/support-center/src/backend/Program.cs

```diff
@@ -12,21 +12,37 @@
 
 // Allow any CORS origin if in DEV
 const string AllowDebugOriginPolicy = "AllowDebugOrigin";
+const string AllowOriginPolicy = "AllowOrigin";
 if (builder.Environment.IsDevelopment())
 {
     builder.Services.AddCors(options =>
     {
         options.AddPolicy(AllowDebugOriginPolicy, builder =>
         {
             builder
-            //.WithOrigins("http://localhost:3000", "http://localhost:3001") // client url
-            .AllowAnyOrigin()
+            .WithOrigins("http://localhost:3000", "http://localhost:3001") // client urls
             .AllowAnyHeader()
             .AllowAnyMethod()
             .AllowCredentials();
         });
     });
 }
+else
```

Comment:
What do you think if we allow all origins in code, and we control CORS in the platform (ex. ACA)?